### PR TITLE
Integrations snapshot

### PR DESCRIPTION
**Pull request by myssynglynx**
- created `useSnapshot()` hook
- created Integrations tab in Settings to connect Snapshot spaces to an Org (needs icon, used the wrench as a placeholder)
- added 'Export to Snapshot' button & cancellation to Edit Task Proposal
- in selected proposal, added button that links to exported proposal

### CODE REVIEW

File: wondrous-app/components/Common/Task/modal.tsx [hunk 1]
```diff
@@ -986,6 +1034,18 @@ export const TaskViewModal = (props: ITaskListModalProps) => {
     }
   };
 
+  // assemble Snapshot URL & open window w/ proposal
+  const openSnapshot = async () => {
+    try {
+      const space = snapshot.key
+      const proposal = fetchedTask?.snapshotProposal
+      const url = `https://${isTest && 'demo'}.snapshot.org/#/${space}/proposal/${proposal}`
+      window.open(url)
```
Comment (andywong418) [inline wondrous-app/components/Common/Task/modal.tsx:1043]: Would this open in a new tab?

File: wondrous-app/components/CreateEntity/createEntityModal.tsx [hunk 2]
```diff
@@ -475,7 +475,11 @@ const CreateLayoutBaseModal = (props) => {
   });
   const [createBounty, { loading: createBountyLoading }] = useMutation(CREATE_BOUNTY);
 
-  const [createTaskProposal, { loading: createTaskProposalLoading }] = useMutation(CREATE_TASK_PROPOSAL);
+  const [createTaskProposal, { loading: createTaskProposalLoading }] = useMutation(CREATE_TASK_PROPOSAL, {
+    onCompleted: data => {
+      console.log('Task created')
```
Comment (andywong418) [inline wondrous-app/components/CreateEntity/createEntityModal.tsx:480]: Let's not log on complete :)

File: wondrous-app/services/snapshot/index.tsx [hunk 3]
```diff
@@ -0,0 +1,362 @@
...
+
+  // update Task Proposal's snapshot_proposal field
+  const [updateSnapshotProposal, { loading: updateSnapshotProposalLoading }] = useMutation(SET_SNAPSHOT_PROPOSAL, {
+    onError: error => {
+      console.error(`error updateSnapshotProposal(): ${error}`);
+    }
+  });
+
+  // creates transaction to export proposal to snapshot, throws error if exporter doesn't have credentials or invalid proposal
+  // HAVING ISSUES, trying to solve w/ Snapshot team
```
Comment (andywong418) [inline wondrous-app/services/snapshot/index.tsx:188]: Is this still a problem?

File: wondrous-app/utils/constants.ts [hunk 4]
```diff
@@ -65,6 +65,7 @@ export const PAYMENT_STATUS = {
 // Supported Chains (ETHEREUM, POLYGON)
 const SUPPORTED_CHAINS = {
   1: 'ETH',
+  3: 'ETH', // REMOVE BEFORE COMMIT
```
Comment (andywong418) [inline wondrous-app/utils/constants.ts:68]: Should this be removed?

**Review by andywong418 — COMMENTED** [review]
@myssynglynx great stuff! Couple of thoughts:
- Let's solve merge conflicts first
- Think there's a build issue (test `yarn build` and you should be able to see the problem)

File: wondrous-app/components/Common/Task/modal.tsx (hunk 1 above)
Comment (myssynglynx) [reply wondrous-app/components/Common/Task/modal.tsx:1043]: yes -- it opens up the proposal on snapshot's website

fyi the `isTest` variable is returned from the `useSnapshot` hook to reflect whether the snapshot being queried is on testnet or mainnet

File: wondrous-app/components/CreateEntity/createEntityModal.tsx (hunk 2 above)
Comment (myssynglynx) [reply wondrous-app/components/CreateEntity/createEntityModal.tsx:480]: This was for debugging-- removed

File: wondrous-app/services/snapshot/index.tsx (hunk 3 above)
Comment (myssynglynx) [reply wondrous-app/services/snapshot/index.tsx:188]: Forgot to get rid of that comment, that was for the snippets I provided to snapshot

File: wondrous-app/utils/constants.ts (hunk 4 above)
Comment (myssynglynx) [reply wondrous-app/utils/constants.ts:68]: was a placeholder to get proposal to work with my test snapshot space, which was on Ropsten -- removed

File: wondrous-app/components/Common/Task/modal.tsx (hunk 1 above)
Comment (andywong418) [reply wondrous-app/components/Common/Task/modal.tsx:1043]: Ah awesome!

**Pull request closed without merging**